### DISCUSSION TITLE
Copy and paste songs to library / playlists

### DISCUSSION THREAD

**Discussion by EduMenges** [ciderapp/Cider#1103] [Feature Request]
Another thing that Spotify does that Apple doesn't is the ability to copy and paste songs, which allow you to quickly select songs from long albums and them to playlists, which is helpful not only when organizing a playlist (with the "cut" feature), but when selecting certain songs from a big album.

![Spotify_eiMNYhVDPG](https://user-images.githubusercontent.com/50274155/171442994-5eed8365-1960-4e2b-a463-2e5fe9238839.gif)

So this another thing that I propose, to make the process of creating playlists less painful.

**Comment by jasoon-chen** [comment]
I just added this feature request in one of my pull requests ([#1181](https://github.com/ciderapp/Cider/pull/1181)). I also really like this feature within spotify and wanted to incorporate it within Cider.

**Reply by EduMenges** [reply]
Very cool, Jaxon! Maybe you could also implement the [drag 'n' drop feature](https://github.com/ciderapp/Cider/discussions/1102)?
